### ISSUE TITLE
Cannot clear console pre-start

### ISSUE THREAD

**Issue by markjackson02** [remy/nodemon#1623]
- `nodemon -v`: 1.18.9
- `node -v`: 12.13.0
- Operating system/terminal environment: Windows 7
- Command you ran: `nodemon server.js`

### Expected behaviour
Console is cleared upon each nodemon restart

### Actual behaviour
Console is not cleared

### Steps to reproduce
I followed the [FAQ recommendation](https://github.com/remy/nodemon/blob/master/faq.md#how-to-clear-the-console-on-restart) to use event start (I tried both package.json nodemonConfig and nodemon.json).

**Comment by remy** [comment]
Can you share your `nodemon.json` file?

**Comment by markjackson02** [comment]
@remy I copied directly from the faq: 

```{
  "events": {
    "start": "cls || clear"
  }
}```

**Comment by remy** [comment]
Which flavour of Windows command are you running too? I know there's quite a few, so it's useful to get it pinned down to help to replicate. If you have a screenshot, this is also very useful.

**Comment by markjackson02** [comment]
What do you mean "flavour of Windows command"? Here is a screenshot. The yarn run alias is to `nodemon server.js` 
![image](https://user-images.githubusercontent.com/35121685/67785904-5dcdcf80-fa44-11e9-85ac-f2c9d256e55a.png)

**Comment by remy** [comment]
Going by the screenshot, I'm guessing that's WSL (rather than git-bash, command prompt, powershell or the like).

And if you try to run the following command in that same terminal (whilst nodemon _isn't_ running): `cls || clear` what do you get?

**Comment by markjackson02** [comment]
It is git-bash. When I run `cls || clear` it clear the console as expected.

**Comment by remy** [comment]
Hmm, I wonder if yarn is spawning inside a different shell and then nodemon is spawning another inside of that…

Is there a command in windows that lets you know the shell features? What we're trying to rule out is:

1. Is the start event firing and running?
2. Is the exec'd command perhaps failing?

To check (1) you could change to this:

```json
{
  "events": {
    "start": "echo 'THE NODEMON IS RESTARTING'"
  }
}
```

If you see the echo message, it's definitely running so the `cls` is the source of the issue, if not, then it's not spawning correctly…

**Comment by markjackson02** [comment]
It echoes correctly: 
![image](https://user-images.githubusercontent.com/35121685/67995575-79da9800-fc21-11e9-8120-f8c354ce7e12.png)

I played around with it, and the `clear` function itself isn't behaving as I'd expect. So it is not an issue with nodemon. 

When I run `clear`, it just scrolls down so my command line is at the top of the screen, and I can scroll up and see my previous commands. Same thing with `reset`. 

It seems to be a known issue with Windows and Git bash. https://github.com/Eugeny/terminus/issues/28

Closing this since it's not a nodemon issue. Thanks!
